src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null,
```

Comment:
As I said before, don't render this component if there is no user selected
```suggestion
  user: User,
```